res2df/hook_implementations/forward_model_steps.py

```diff
@@ -0,0 +1,138 @@
+import shutil
+
+try:
+    from ert import (  # type: ignore
+        ForwardModelStepDocumentation,
+        ForwardModelStepJSON,
+        ForwardModelStepPlugin,
+        ForwardModelStepValidationError,
+    )
+    from ert import plugin as ert_plugin  # type: ignore
+except ModuleNotFoundError:
+    # ert is not installed, use dummy/transparent function decorator:
+    def ert_plugin(name: str = ""):
+        def decorator(func):
+            return func
+
+        return decorator
+
+    class ForwardModelStepPlugin:  # type: ignore
+        pass
+
+    class ForwardModelStepJSON:  # type: ignore
+        pass
+
+    class ForwardModelStepDocumentation:  # type: ignore
```

Comment:
This could have been swapped with`ForwardModelStepPlugin` to get perfect ordering

Reply:
we like order